tracing/tck/README.adoc

```diff
@@ -56,15 +56,43 @@ You also need to specify which tests you want to run, e.g. create a file `tck-su
 ----
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
-<suite name="microprofile-opentracing-TCK" verbose="2" configfailurepolicy="continue" >
-
-    <test name="microprofile-telemetry 1.0 TCK">
+<suite name="microprofile-telemetry-tracing-TCK" verbose="2" configfailurepolicy="continue" >
+    <test name="telemetry-tracing-tests" verbose="10">
+        <groups>
+            <run>
+                <!-- The required REST and CDI tests that all MP-TELEMETRY implementations must pass.-->
+                <include name="cdi"/>
+                <include name="rest"/>
+            </run>
+        </groups>
         <packages>
-            <package name="org.eclipse.microprofile.telemetry.tracing.tck.*">
-            </package>
+            <package name="org.eclipse.microprofile.telemetry.tracing.tck.*" />
         </packages>
     </test>
+</suite>
+----
 
+There are optional tests for the B3 and Jaeger progagation formats. These tests are present in the `optional-tests` group. A MP-Telemetry implementation is not required to pass these tests in order to be considered a valid implementation. If you wish to run them, include the group in your `tck-suite.xml`. 
+
+[source, xml]
+----
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="microprofile-telemetry-tracing-TCK" verbose="2" configfailurepolicy="continue" >
+    <test name="telemetry-tracing-tests" verbose="10">
+        <groups>
+            <run>
+                <!-- The required REST and CDI tests that all MP-TELEMETRY implementations must pass -->
+                <include name="cdi"/>
```

Comment:
I don't think you defined these groups? Can you use exclude?

Reply:
This was from an older commit. I have squashed them to avoid confusion